test/test_functions

```diff
@@ -1983,7 +2026,7 @@ purge_disk_cache() {
 try_automount() {
```

Comment:
Hmmm that's not exactly what happens, I think. On Mac OS X there is no automount, so we have to manually mount it to cause **the most similar effect**. Of course this is not a real automount, and nothing guarantees that (for linux or mac) it's going to be either "mounted" or "automounted", depending on the OS. In both cases if it fails for whatever reason it does nothing, which is exactly what it's supposed to do.

So I see that the function is "trying to trigger the automount" cheating a bit on mac, but essentially doing that.